model_downloader/list_topologies.yml

```diff
@@ -1097,6 +1097,29 @@ topologies:
     framework: mxnet
     license: https://github.com/lachinov/brats2018-graphlabunn/blob/master/LICENSE
 
+  - name: "ssd-mobilenet-v2-0.35.1.-barrier-256x256-0123"
```

Comment:
Let's use the name `vehicle-license-plate-detection-barrier-0123"

No need to change the name of the tarball.